addon/components/instance-link.hbs

```diff
@@ -0,0 +1,18 @@
+{{#if @links.length}}
+    <div class="uk-flex uk-flex-inline uk-flex-middle">
+    <UkIcon @icon="link" class="uk-text-emphasis" />
+    {{#each @links as |link index|}}
+        {{#if index}}
```

Comment:
i think this also appends a `,` at the last element?

Reply:
It prepends a comma, unless it is the first element.